app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -19,6 +19,13 @@ def suggested_conditions
     def submit
       form_content = JSON.parse(request.body.string)
 
+      # TODO: While testing `all claims` submissions we will be merging the submission
+      # with a hard coded "completed" form which will gap fill any missing data. This should
+      # be removed before `all claims` goes live.
+      form_content = all_claims_integration(form_content) if form_content.key?('21-526EZ-all-claims')
```

Comment:
Theres no other way to identify whether a submission is `all claims` or `claims for increase only` besides changing the base key when submitting the form. `21-526EZ-all-claims` has been suggested by Mark but we should agree on this key as a group.